lib/generators/provider/templates/spec/spec_helper.rb

```diff
@@ -3,11 +3,10 @@
   SimpleCov.start
 end
 
-# Uncomment in case you use vcr cassettes
-# VCR.configure do |config|
-#   config.ignore_hosts 'codeclimate.com' if ENV['CI']
-#   config.cassette_library_dir = File.join(ManageIQ::Providers::<%= class_name %>::Engine.root, 'spec/vcr_cassettes')
-# end
+VCR.configure do |config|
+  config.ignore_hosts 'codeclimate.com' if ENV['CI']
+  config.cassette_library_dir = File.join(ManageIQ::Providers::<%= class_name %>::Engine.root, 'spec/vcr_cassettes')
+end
```

Comment:
It just makes is more hassle to re-run the generator across all repos and most of them have cassettes enabled.

But gotcha, I'll make it an option